Formula/molden.rb

```diff
@@ -0,0 +1,20 @@
+class Molden < Formula
+  desc "Pre- and post-processing of molecular and electronic structure"
+  homepage "http://www.cmbi.ru.nl/molden/"
+  url "ftp://ftp.cmbi.ru.nl/pub/molgraph/molden/molden5.7.tar.gz"
+  sha256 "5e7b3a8bf9251626d362b1d1dd8e4362054e6bce505c43e10254265857bb6e7d"
+  revision 1
+
+  depends_on :fortran
+  depends_on :x11
+
+  def install
+    system "make"
+    bin.install "molden", "gmolden"
```

Comment:
Needs a `make install` to be considered for inclusion.

Reply:
The makefile uses `sudo` and hardcodes `/usr/local/bin`. I've reported it by email, but it seems cleaner to install binaries than to patch.

Reply:
OK, I think we'll need to pass on this PR then.

Reply:
1227 formulas have `bin.install`, and it's not only legacy stuff. We regularly accept new formulas with manual install for Rust and other non make-based projects. Many of our top formulas use it, such as go.

Reply:
This uses make, and needs a functional make install to be included. The way cargo works is irrelevant.